app/models/duty_option_result.rb

```diff
@@ -13,4 +13,18 @@ class DutyOptionResult
   attribute :warning_text
   attribute :order_number
   attribute :geographical_area_description
+
+  def footnote
+    (super + footnote_suffix).html_safe
+  end
+
+  def footnote_suffix
+    @footnote_suffix.presence || ''
```

Comment:
Maybe mark it as html_safe here instead? `(@footnote_suffix.presence || '').html_safe`